covalent_slurm_plugin/slurm.py

```diff
@@ -541,6 +558,9 @@ async def run(self, function: Callable, args: List, kwargs: Dict, task_metadata:
         node_id = task_metadata["node_id"]
         results_dir = task_metadata["results_dir"]
         task_results_dir = os.path.join(results_dir, dispatch_id)
+        self._current_remote_workdir = os.path.join(
```

Comment:
I think this is really the only method that will end up using `current_remote_workdir` so you might want to make it a local variable here.

Reply:
It's used in a few places, so I've kept it under `self._`. Happy to adjust if you feel it necessary.